benchmarks/report_bayesmark.py

```diff
@@ -0,0 +1,92 @@
+from collections import Counter
+import io
+import json
+import os
+from typing import Dict
+
+
+_TABLE_HEADER = "\n|Ranking|Solver|Score|\n|---:|:---|---:|\n"
```

Comment:
Hmm... I see. Then does that mean that the line in the output of plot does not correspond to this score? If they correspond, I was wondering if it would be possible to get the mean and standard deviation based on the plot.

Reply:
It does correspond (score seems to be [`100 * (1 - normalized mean)`](https://github.com/uber/bayesmark/blob/master/docs/scoring.rst#mean-scores)), however error bars on that plot do not show standard deviation, but rather are based on  [t-statistic](https://github.com/uber/bayesmark/blob/master/docs/scoring.rst#error-bars). Also those numbers are kept as precalculated aggregates (mean, lower and upper bound for each optimizer at any step), hence my comment about re-writing the aggregation mechanism to get additional stats (which might not be a bad idea).

Reply:
Thanks for the explanation. I think we can re-write the aggregation mechanism as a follow-up task.